Dockerfile

```diff
@@ -16,7 +16,9 @@ RUN pip3 install /home/appuser/document-to-podcast
 RUN python3 demo/download_models.py
```

Comment:
Ok I think the problem might be here. 
Inside this script, HuggingFace is downloading the models to `~/.cache` here but at this point `~` doesn't point to `/home/appuser`. 

I think if we set `USER appuser` before running `demo/download_models.py` the error should be fixed

Reply:
that makes sense :) I'll do that and test! thanks David!